packages/victory-candlestick/src/index.d.ts

```diff
@@ -0,0 +1,94 @@
+// Definitions by: Alexey Svetliakov <https://github.com/asvetliakov>
+//                 snerks <https://github.com/snerks>
+//                 Krzysztof Cebula <https://github.com/Havret>
+//                 Vitaliy Polyanskiy <https://github.com/alreadyExisted>
+//                 James Lismore <https://github.com/jlismore>
+//                 Stack Builders <https://github.com/stackbuilders>
+//                 Esteban Ibarra <https://github.com/ibarrae>
+//                 Dominic Lee <https://github.com/dominictwlee>
+//                 Dave Vedder <https://github.com/veddermatic>
+//                 Alec Flett <https://github.com/alecf>
+
+import * as React from "react";
+import {
+  EventPropTypeInterface,
+  OrientationTypes,
+  OriginType,
+  StringOrNumberOrCallback,
+  VictoryCommonProps,
+  VictoryDatableProps,
+  VictoryStyleObject,
+  VictoryLabableProps,
+  VictoryMultiLabeableProps,
+  VictoryStyleInterface
+} from "victory-core";
+
+export interface VictoryCandlestickStyleInterface extends VictoryStyleInterface {
+  closeLabels?: VictoryStyleObject;
+  data?: VictoryStyleObject;
+  highLabels?: VictoryStyleObject;
+  labels?: VictoryStyleObject;
+  lowLabels?: VictoryStyleObject;
+  openLabels?: VictoryStyleObject;
+  parent?: VictoryStyleObject;
+}
+
+export type VictoryCandlestickLabelsType = (string | number)[] | boolean | ((datum: any) => number);
+
+export interface VictoryCandlestickProps
+  extends VictoryCommonProps,
+    VictoryDatableProps,
+    VictoryLabableProps,
+    VictoryMultiLabeableProps {
+  candleColors?: {
+    positive?: string;
+    negative?: string;
+  };
+  candleRatio?: number;
+  candleWidth?: number | Function;
+  close?: StringOrNumberOrCallback | string[];
+  closeLabelComponent?: React.ReactElement;
+  closeLabels?: VictoryCandlestickLabelsType;
+  eventKey?: StringOrNumberOrCallback | string[];
+  events?: EventPropTypeInterface<
+    | "data"
+    | "labels"
+    | "open"
+    | "openLabels"
+    | "close"
+    | "closeLabels"
+    | "low"
+    | "lowLabels"
+    | "high"
+    | "highLabels",
+    StringOrNumberOrCallback | string[]
+  >[];
+  high?: StringOrNumberOrCallback | string[];
+  highLabelComponenet?: React.ReactElement;
+  highLabels?: VictoryCandlestickLabelsType;
+  labelOrientation?: {
```

Comment:
This should also be able to take a single value

Reply:
thanks for catching this. I think when I refactored it, I accidentally made some wrong changes. When I looked at the docs, https://formidable.com/open-source/victory/docs/victory-candlestick#labelorientation, it says it can take a string or an object with an option for some or all of the labels. 
Is this the correct way to type it out? I am unsure if the `labels` props belongs in the object, I was getting errors when I didn't include it. I am not sure if the demo has the wrong key here.  https://github.com/FormidableLabs/victory/blob/master/demo/js/components/victory-candlestick-demo.js#L134

```labelOrientation?:
    | OrientationTypes
    | {
        open?: OrientationTypes;
        close?: OrientationTypes;
        labels?: OrientationTypes;
        low?: OrientationTypes;
        high?: OrientationTypes;
      };```

Reply:
I think `labels` _is_ included here and your demo is correct, in case the user wants the key to display a particular label, if I'm reading the src correctly? In that case, though, would it need an `OrientationTypes` or just a `string`?

Reply:
**Wait**! Is this prop orienting a label in a particular direction? Is this the default prop or something? 🤔  I agree now that it seems odd